src/main/webapp/onelive/archive/functions.js

```diff
@@ -30,3 +30,11 @@ function loadYoutubeVideos() {
         }
     });
 }
+
+function loadYoutubeVideoPlayer(videoTag){
+        if (videoTag != null){
```

Comment:
Shall we use the same name from the API? `videoId`

Reply:
By the way, Is it possible to have a null value?

Reply:
No, I'll remove that.